From f56d36a2323f04bb4936a106efdcf8c4754e2233 Mon Sep 17 00:00:00 2001
From: Anand Balagopalakrishnan <anandb@ti.com>
Date: Sun, 24 Jan 2016 14:16:10 +0530
Subject: [PATCH 1/1] configure: disable kmscube

kmscube support requires proprietary EGL extensions EGL_RAW_VIDEO_TI_DMABUF
which is not supported in DDK 1.14.

Future versions of DDK will support the standard EGL extension
EXT_image_dma_buf_import which is ratified by Khronos. This will require
corresponding changes to the application.

Till then, disable kmscube in omapdrmtest.

Signed-off-by: Anand Balagopalakrishnan <anandb@ti.com>
---
 configure.ac |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/configure.ac b/configure.ac
index c026733..05180a6 100644
--- a/configure.ac
+++ b/configure.ac
@@ -51,7 +51,7 @@ AM_CONDITIONAL(ENABLE_V4L2_DMABUF, [test "x$HAVE_V4L2_DMABUF" = xyes])
 
 # Check optional KMSCUBE:
 AC_ARG_ENABLE([kmscube], AS_HELP_STRING([--disable-kmscube], [disable kmscube display support]))
-AS_IF([test "x$enable_kmscube" != "xno"], [PKG_CHECK_EXISTS(gbm egl glesv2, [HAVE_KMSCUBE=yes], [HAVE_KMSCUBE=no])])
+HAVE_KMSCUBE="no"
 if test "x$HAVE_KMSCUBE" = "xyes"; then
 	AC_DEFINE(HAVE_KMSCUBE, 1, [Have KMSCUBE support])
 	PKG_CHECK_MODULES(GBM, gbm)
-- 
1.7.9.5

